core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -46,16 +46,34 @@ object ZkUtils {
   val ClusterIdPath = "/cluster/id"
   val BrokerIdsPath = "/brokers/ids"
   val BrokerTopicsPath = "/brokers/topics"
-  val ControllerPath = "/controller"
-  val ControllerEpochPath = "/controller_epoch"
   val ReassignPartitionsPath = "/admin/reassign_partitions"
   val DeleteTopicsPath = "/admin/delete_topics"
   val PreferredReplicaLeaderElectionPath = "/admin/preferred_replica_election"
   val BrokerSequenceIdPath = "/brokers/seqid"
-  val IsrChangeNotificationPath = "/isr_change_notification"
-  val EntityConfigPath = "/config"
   val EntityConfigChangesPath = "/config/changes"
 
+  // Important: it is necessary to add any new persistent Zookeeper path here
+  val AdminPath = "/admin"
+  val BrokersPath = "/brokers"
+  val ClusterPath = "/cluster"
```

Comment:
Could we define the top level constants earlier and reference them in subpaths like "/cluster/id"?